packages/boilerplate/server/webpack.config.babel.js

```diff
@@ -40,17 +40,15 @@ export default (env) => {
       strictExportPresence: true, // If you import something that isn't exported
       rules: [
         {
-          test: /\.js$/,
-          exclude: /node_modules/,
+          test: /\.jsx?$/,
+          // Skip node_modules, with the exception of packages in the monorepo
+          exclude: /[\\/]node_modules[\\/](?!@respond-framework)/,
```

Comment:
remember, take advantage of webpack-merge. We wanna be inheriting as much webpack composable as possible if we are not already.  slack me if you want an example, i build Icarus webpack builds to be mergable

Reply:
Is there an advantage of `webpack-merge` over this kind of method with parameters?